LICENSE

```diff
@@ -1,17 +1,15 @@
-#######
-License
-#######
+check_po, python package to automate task on traduction file .po
+Copyright (C) 2016  aRkadeFR
```

Comment:
plus the real author in package metadata: https://github.com/novafloss/check_po/blob/master/setup.cfg#L14
Which FMPOV is the reference.